src/style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
```

Comment:
Do not apply any styles to the `*` selector except `box-sizing` move it to the `hmtl` tag selector